webpack/clientNoReact.js

```diff
@@ -20,6 +20,12 @@ export default merge({}, clientWebpackConfig, {
       commonjs: 'react-dom',
       amd: 'react-dom'
     },
+    'prop-types': {
```

Comment:
is this necessary ? React proptypes should be stripped from any production  build. We don't want to execute PropTypes code in production

Reply:
The tree-shaking is not that perfect... the `prop-types` module is still referenced in production builds.